infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -38,26 +38,44 @@ nightly_builds = [
   }
 ]
 
-# TODO: Remove this after the 2.1 release
-xrt_versioned_builds = [
+# Built on push to specific tag.
+versioned_builds = [
+  # Remove libtpu from PyPI builds
   {
-    accelerator    = "tpu"
-    python_version = "3.10"
-    pytorch_git_rev = "v2.1.0"
-    package_version = "2.1.0+xrt"
+    git_tag         = "v2.2.0-rc1"
```

Comment:
Hi, @zpcore, thanks for the update, not sure how PyTorch rag 2.2 at branch cut day, in 2.1 release, PyTorch's wheel version is `2.1.0-rc0` at the branch cut day; if we want to catch and start testing 2.2 release from branch cut day, do we want to use "v2.2.0-rc0"?

Reply:
Hi @ManfeiBai , I checked the pytorch repo but didn't find tag end with `-rc0`. Can you help double check if we should start from `-rc0`? They may delete it after the release. Thanks,

Reply:
Thanks for confirming this, @zpcore, you are right, I didn't find rc0, we could start from `-rc1` based on 2.1.0 release history